BooruSharp/Booru/ABooru.cs

```diff
@@ -117,27 +117,27 @@ private protected virtual async Task<IEnumerable> GetTagEnumerableSearchResultAs
         /// <summary>
         /// Gets a value indicating whether http:// scheme is used instead of https://.
         /// </summary>
-        protected bool UsesHttp => _options.HasFlag(BooruOptions.UseHttp);
+        public bool UsesHttp => _options.HasFlag(BooruOptions.UseHttp);
 
         /// <summary>
         /// Gets a value indicating whether tags API uses XML instead of JSON.
         /// </summary>
-        protected bool TagsUseXml => _options.HasFlag(BooruOptions.TagApiXml);
+        public bool TagsUseXml => _options.HasFlag(BooruOptions.TagApiXml);
 
         /// <summary>
         /// Gets a value indicating whether comments API uses XML instead of JSON.
         /// </summary>
-        protected bool CommentsUseXml => _options.HasFlag(BooruOptions.CommentApiXml);
+        public bool CommentsUseXml => _options.HasFlag(BooruOptions.CommentApiXml);
```

Comment:
Not sure we want these exposed to the user

Reply:
True, not that one probably. I mainly wanted NoMoreThanTwoTags to avoid the exception with danbooru.

Reply:
Where are you using it? If it's only in the library but outside of ABooru you can use internal

Reply:
To clarify, in my own program, I'm using the same code to get posts from all the boorus but any booru with this flag set will cause an exception if I happen to have more than 2 tags. So I need some way to check for it that isn't just `type is Danbooru`.

Reply:
Ah makes sence
I think we can keep the more than 2 tags public then, for the others since it's really internal use I think they can go back to protected